buildspec.yml

```diff
@@ -12,7 +12,7 @@ phases:
     commands:
       - echo Build started on `date`
       - echo Building the Docker image...
-      - docker build -t $IMAGE_REPO_NAME:$IMAGE_TAG .
+      - docker build -t $IMAGE_REPO_NAME:$IMAGE_TAG -f docker/web/Dockerfile
```

Comment:
```suggestion
      - docker build -t $IMAGE_REPO_NAME:$IMAGE_TAG -f docker/web/Dockerfil e.
```

Reply:
You sure about this suggestion ? It feels to be it would be a mistake

Reply:
Yeap, sorry. Updated.